media/css/m24/vars/_color.scss

```diff
@@ -2,40 +2,43 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
+@use '~@mozilla-protocol/core/protocol/css/includes/lib' as lib;
```

Comment:
👍 for namespacing, I think by default it will use the final part of the path, so "as lib" is redundant in this case but it doesn't hurt to be explicit